logic/scorer/scorer_test.go

```diff
@@ -10,199 +11,187 @@ import (
 	"github.com/joshprzybyszewski/cribbage/model"
 )
 
-func TestPointsStandardFunThings(t *testing.T) {
-	testCases := []struct {
+func BenchmarkHandPoints(b *testing.B) {
+	hand := randomHand(b, 5)
```

Comment:
:cactus: why not create a _ton_ of random hands and then score all of them in the benchmark?

Reply:
:bulb: 
```go
rand.Seed(69)
	var hands [][]model.Card

	for i := 0; i < 1000; i++ {
		hands = append(hands, randomHand(b, 5))
	}

	b.Run(`scoring random hand`, func(b *testing.B) {
		b.ResetTimer()
		for i := 0; i < b.N; i++ {
			for _, hand := range hands {
				s := HandPoints(hand[0], hand[1:])
				require.Less(b, s, 30)
				require.GreaterOrEqual(b, s, 0)
			}
		}
	})
```